translate/api/Tests/TestBatchTranslate.cs

```diff
@@ -0,0 +1,72 @@
+﻿using System;
+using System.IO;
+using System.Collections.Generic;
+using System.Text;
+using Xunit;
+using Google.Cloud.Translate.V3.Samples;
+using GoogleCloudSamples;
+using Google.Cloud.Storage.V1;
+using Google.Apis.Storage.v1.Data;
+using System.Linq;
+
+namespace Tests
+{
+    public class BatchTranslateTest : IDisposable
+    {
+        protected string ProjectId { get; private set; } = Environment.GetEnvironmentVariable("GOOGLE_PROJECT_ID");
+        protected string _bucketName { get; private set; }
+        protected Bucket Bucket { get; private set; }
+        protected string InputUri { get; private set; } = "gs://cloud-samples-data/translation/glossary_ja.csv";
+        readonly CommandLineRunner _sample = new CommandLineRunner()
+        {
+            VoidMain = TranslateV3BatchTranslateTextMain.Main
+        };
+
+        // Setup
+        public BatchTranslateTest()
+        {
+            // Create temp bucket
+            var storageClient = StorageClient.Create();
+            _bucketName = "translate-v3-" + TestUtil.RandomName();
+            Bucket = storageClient.CreateBucket(ProjectId, _bucketName);
+        }
+
+        public void Dispose()
+        {
+            var storageClient = StorageClient.Create();
+
+            // Clean up output files.
+            var blobList = storageClient.ListObjects(_bucketName, "");
+            foreach (var outputFile in blobList.Where(x => x.Name.Contains("translation/")).Select(x => x.Name))
+            {
+                storageClient.DeleteObject(_bucketName, outputFile);
+            }
+            storageClient.DeleteBucket(_bucketName);
+        }
+
+        /// <summary>
+        ///  Run the command and track all cloud assets that were created.
+        /// </summary>
+        /// <param name="arguments">The command arguments.</param>
+        public ConsoleOutput Run(params string[] arguments)
+        {
+            return _sample.Run(arguments);
+        }
+
+        [Fact]
+        public void TestBatchTranslateText()
```

Comment:
Use Test as a suffix, not as a prefix.

Reply:
Should I add TranslateV3 prefix for all tests methods?
![image](https://user-images.githubusercontent.com/45373284/67918914-92807a80-fb5b-11e9-9b5d-89fcbe04c55c.png)

Reply:
Or take Test suffix out of all Facts?

Reply:
No, please don't add TransalateV3.
You can name the classes TransalateTextTests (Tests in plural) and the individual test TransalateTextTest (Test in singular).